### PR TITLE
CSD01 second try

### PR DESCRIPTION
**Pull request by ralfhandl**
This PR provides a new build and new ZIP files for CSD01.

- [x] revert #253
- [x] merge #257 
- [x] build MD, HTML, PDF, and ZIP

After approval
- [x] delete tag [core/V4.02_CSD01](https://github.com/oasis-tcs/odata-specs/releases/tag/core%2FV4.02_CSD01)
- [x] tag last commit in this branch as `core/V4.02_CSD01`
- [x] delete temporary tag [core/V4.02_CSD01#2](https://github.com/oasis-tcs/odata-specs/releases/tag/core%2FV4.02_CSD01%232)
- [x] close this PR without merging and delete the branch once the public review has started

After public review
- [x] new entries in [Published documents](https://github.com/oasis-tcs/odata-specs/blob/main/README.md#published-documents) table

Feedback from TC Admin
-  [x] URLs for Protocol and URL Conventions are missing the `/partx-yyy` path segment, see [TCADMIN-4479](https://issues.oasis-open.org/browse/TCADMIN-4479?focusedCommentId=84378&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-84378), seems to affect only the `0 frontmatter.md` files
- [x] Remove "Latest stage" for additional artifacts in CSDL JSON and CSDL XML
- [x] XML Namespaces only for CSDL XML 
- [x] Backport to `main` branch, also adjust "Previous stage" URLs there

### CODE REVIEW

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

File: odata-csdl/json.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 subtitle: Committee Specification Draft 01
 dirname: odata-csdl-json
-filename: odata-csdl-json-v4.02-csd02
-stage: csd02
-previousStage: csd01
+filename: odata-csdl-json-v4.02-csd01
+stage: csd01
+previousStage: n/a
 pubdate: 28 February 2024
 pubdateISO: '2024-02-28'
```
Comment (HeikoTheissen) [inline odata-csdl/json.yaml:8]: Must this be changed to 2024-03-06 in all metadata files?

Comment (HeikoTheissen) [reply odata-csdl/json.yaml:8]: According to Paul Knight, no.

**Review by HeikoTheissen — DISMISSED** [review]

File: odata-protocol/0 frontmatter.md [hunk 2]
```diff
@@ -58,6 +58,10 @@ This specification is related to:
 * _OData Data Aggregation Extension Version 4.0_. Edited by Ralf Handl, Hubert Heijkers, Gerald Krause, Michael Pizzo, Heiko Theißen, and Martin Zurmuehl. Latest stage: https://docs.oasis-open.org/odata/odata-data-aggregation-ext/v4.0/odata-data-aggregation-ext-v4.0.html
 * _OData Extension for Temporal Data Version 4.0_. Edited by Ralf Handl, Hubert Heijkers, Gerald Krause, Michael Pizzo, Heiko Theißen, and Martin Zurmuehl. Latest stage: https://docs.oasis-open.org/odata/odata-temporal-ext/v4.0/odata-temporal-ext-v4.0.html
 
+#### Declared XML namespaces:
+* http://docs.oasis-open.org/odata/ns/edmx
+* http://docs.oasis-open.org/odata/ns/edm
+
```
Comment (ralfhandl) [inline odata-protocol/0 frontmatter.md:64]: I think we need these only for CSDL XML, we don't have them in the 4.01 documents https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html

File: odata-url-conventions/0 frontmatter.md [hunk 3]
```diff
@@ -57,6 +57,10 @@ This specification is related to:
 * _OData Data Aggregation Extension Version 4.0_. Edited by Ralf Handl, Hubert Heijkers, Gerald Krause, Michael Pizzo, Heiko Theißen, and Martin Zurmuehl. Latest stage: https://docs.oasis-open.org/odata/odata-data-aggregation-ext/v4.0/odata-data-aggregation-ext-v4.0.html
 * _OData Extension for Temporal Data Version 4.0_. Edited by Ralf Handl, Hubert Heijkers, Gerald Krause, Michael Pizzo, Heiko Theißen, and Martin Zurmuehl. Latest stage: https://docs.oasis-open.org/odata/odata-temporal-ext/v4.0/odata-temporal-ext-v4.0.html
 
+#### Declared XML namespaces:
+* http://docs.oasis-open.org/odata/ns/edmx
+* http://docs.oasis-open.org/odata/ns/edm
+
```
Comment (ralfhandl) [inline odata-url-conventions/0 frontmatter.md:63]: Remove, only needed for CSDL XML

File: odata-json-format/0 frontmatter.md [hunk 4]
```diff
@@ -52,6 +52,10 @@ This specification is related to:
 * _OData Common Schema Definition Language (CSDL) JSON Representation Version 4.02_. Edited by Michael Pizzo, Ralf Handl, and Heiko Theißen. Latest stage: https://docs.oasis-open.org/odata/odata-csdl-json/v4.02/odata-csdl-json-v4.02.html
 * _OData Common Schema Definition Language (CSDL) XML Representation Version 4.02_. Edited by Michael Pizzo, Ralf Handl, and Heiko Theißen. Latest stage: https://docs.oasis-open.org/odata/odata-csdl-xml/v4.02/odata-csdl-xml-v4.02.html
 
+#### Declared XML namespaces:
+* http://docs.oasis-open.org/odata/ns/edmx
+* http://docs.oasis-open.org/odata/ns/edm
+
```
Comment (ralfhandl) [inline odata-json-format/0 frontmatter.md:58]: Remove, only needed for CSDL XML

File: odata-csdl/0 frontmatter.md [hunk 5]
```diff
@@ -70,6 +70,10 @@ This specification is related to:
 :
 * _OData JSON Format Version 4.02_. Edited by Ralf Handl, Mike Pizzo, and Heiko Theißen. Latest stage: https://docs.oasis-open.org/odata/odata-json-format/v4.02/odata-json-format-v4.02.html
 
+#### Declared XML namespaces:
+* http://docs.oasis-open.org/odata/ns/edmx
+* http://docs.oasis-open.org/odata/ns/edm
+
```
Comment (ralfhandl) [inline odata-csdl/0 frontmatter.md:76]: This needs to be conditional:
- CSDL XML needs the XML namespaces
- CSDL JSON needs the JSON Schema

Comment (HeikoTheissen) [reply odata-csdl/0 frontmatter.md]: But we don't introduce a new namespace in our JSON schema, do we?

Comment (ralfhandl) [reply odata-csdl/0 frontmatter.md]: Correct, I compared with the 4.01 version: only CSDL XML mentions XML namespaces, the JSON Schema is only referenced in Additional Artifacts.

**Pull request closed without merging**